apps/labs/posts/thoughts-on-reviewing-for-accessibility.md

```diff
@@ -0,0 +1,195 @@
+---
+title: 'Checking for accessibility: thoughts and a checklist!'
+author: isabela-presedo-floyd
+published: June 06, 2022
+category: [Accessibility, OSS Experience]
+featuredImage:
+  src: /posts/thoughts-on-reviewing-for-accessibility/jlabaccess4.png
+  alt: Illustration of checkboxes with ticks and checkmarks
+hero:
+  imageSrc: /posts/thoughts-on-reviewing-for-accessibility/hero.jpeg
+  imageAlt: Lorem ipsum dolor
```

Comment:
We'll want to replace this lorem ipsum text evetually.

Reply:
yeah I know, But I'm waiting for the real Hero images. This is a placeholder.